src/app/public/modules/list-view-checklist/list-view-checklist.component.ts

```diff
@@ -368,4 +299,13 @@ export class SkyListViewChecklistComponent extends ListViewComponent implements
       }
     this.disableToolbar(isSelected);
   }
+
+  private showSelectedValuesEqual(prev: ListFilterModel[], next: ListFilterModel[]) {
+    const prevShowSelectedFilter = prev.find(filter => filter.name === 'show-selected');
+    const nextShowSelectedFilter = next.find(filter => filter.name === 'show-selected');
+    if (prevShowSelectedFilter && nextShowSelectedFilter) {
```

Comment:
I can't find it in our style guide; however, I believe @Blackbaud-SteveBrush mentioned once that we should have line breaks on both sides of a block of code like the `if block` here.

Reply:
done.